vividus-tests/src/main/resources/story/integration/ProxyStepsTests.story

```diff
@@ -4,8 +4,8 @@ Meta:
     @proxy
 
 Scenario: Verify step Then number of HTTP $httpMethod requests with URL pattern `$urlPattern` is $comparisonRule `$number`
-Given I am on a page with the URL 'http://httpbin.org/get'
-Then number of HTTP GET requests with URL pattern `http://httpbin.org/get` is EQUAL TO `1`
+Given I am on a page with the URL 'https://httpbin.org/get'
```

Comment:
I propose to switch the whole story to HTTPS

Reply:
we need to have both HTTP and HTTPS to easily debug issues if any